proximal/examples/test_deconv_sinv_psf.py

```diff
@@ -68,8 +82,8 @@ def cost_function(x, im, psf, mu=0.1, using_px_conv=True):
 
 # Solve
 x = px.Variable(im_sb.shape)
-prob = px.Problem(cost_function(x, im_sb, psf=psf2, mu=0.1))
-prob.solve(solver='pc', max_iters=20, verbose=True, x0=im_sb)
+prob = px.Problem(cost_function(x, im_sb, psf=psf2, mu=0.1, using_px_conv=True))
+prob.solve(solver='ladmm', eps_abs=1e-2, verbose=True)
```

Comment:
I recommend `ladmm` solver. In the perfect world, ADMM is better than linearized ADMM because the convolution kernel can be diagonalized in the Fourier domain; closed form solution is available in the lambda update step. But, on my machine, it takes longer for ADMM to deduce the diagonal matrix than to solve it with iterations.

Reply:
Time log: 
```python
Estimated params [lambda = 1.000 | mu = 1.000 | L_est = 1.0000]
iter 0: ||r||_2 = 281.614, eps_pri = 46.620, ||s||_2 = 248.245, eps_dual = 32.652
op = LIN-ADMM iteration, evals = 3, total_time (ms) = 27180.093102972023, avg_time (ms) = 9060.031034324007
prox funcs:
op = group_norm1, evals = 3, total_time (ms) = 4596.789567032829, avg_time (ms) = 1532.263189010943
K forward ops:
op = vstack, evals = 6, total_time (ms) = 1306.382909999229, avg_time (ms) = 217.7304849998715
op = scale, evals = 6, total_time (ms) = 89.03879497665912, avg_time (ms) = 14.83979916277652
op = grad, evals = 6, total_time (ms) = 123.35802800953388, avg_time (ms) = 20.559671334922314
op = copy, evals = 6, total_time (ms) = 30.269926937762648, avg_time (ms) = 5.044987822960441
op = split, evals = 6, total_time (ms) = 247.56074202014133, avg_time (ms) = 41.26012367002355
op = CompGraph, evals = 6, total_time (ms) = 1797.382287972141, avg_time (ms) = 299.56371466202353
K adjoint ops:
op = vstack, evals = 9, total_time (ms) = 1041.787673952058, avg_time (ms) = 115.75418599467311
op = scale, evals = 9, total_time (ms) = 113.09609888121486, avg_time (ms) = 12.566233209023872
op = grad, evals = 9, total_time (ms) = 165.54896999150515, avg_time (ms) = 18.394329999056126
op = copy, evals = 9, total_time (ms) = 32.63302205596119, avg_time (ms) = 3.6258913395512433
op = split, evals = 9, total_time (ms) = 546.3189001311548, avg_time (ms) = 60.70210001457276
op = CompGraph, evals = 9, total_time (ms) = 1900.4220741335303, avg_time (ms) = 211.1580082370589
```

Reply:
Closing this tread because the end user prefers the `pc` solver.